src/BreathOfTheWild/Mods/MasterModeOptions/rules.txt

```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Master Mode Options"
+path = "The Legend of Zelda: Breath of the Wild/Mods/Master Mode Options"
+description = Allows you to pick and choose which of Nintendo's tweaks |are applied to Master Mode (Does nothing in Normal Mode).|A reboot is required to apply changes made while the game |is running. ||Logic and patching by Torphedo and Fleevoid, with lots of |help from Exzap and leoetlino.
+version = 7
+
+[Default]
+$debug:int = 0
+$bit0:int = 0x1
+$bit1:int = 0x0
+$bit3:int = 0x8
+$bit4:int = 0x10
+$bit5:int = 0x20
+$bit6:int = 0x40
+$bit7:int = 0x80
+$bit8:int = 0x100
+
+[Preset]
+name = Enabled (Vanilla)
+category = IsLastPlayHardMode (Does nothing)
+condition = $debug == 1
+$bit0:int = 0x1
+
+[Preset]
+name = Disabled
+category = IsLastPlayHardMode (Does nothing)
+condition = $debug == 1
+$bit0:int = 0x0
+
+[Preset]
+name = Disabled (Vanilla)
+category = Nerf Cooking (Appears to be broken or unimplemented)
+condition = $debug == 1
+$bit1:int = 0x0
+
+[Preset]
+name = Enabled
+category = Nerf Cooking (Appears to be broken or unimplemented)
+condition = $debug == 1
+$bit1:int = 0x2
+
+[Preset]
+name = Disabled (Vanilla)
+category = 1-Hit-Kill Protection
+$bit3:int = 0x8
+
+[Preset]
+name = Enabled
+category = 1-Hit-Kill Protection
+$bit3:int = 0x0
+
+[Preset]
+name = Enabled (Vanilla)
+category = Patch Ganon Stun Lock

```

Comment:
Is there an easier way to name this? I assume it prevents ganon from getting stun locked right? Maybe call it `No Ganon Stun Lock` or something? The original japanese english is not always very easy to understand for users.